R/recipe_commit.R

```diff
@@ -37,7 +37,11 @@ orderly_commit <- function(id, name = NULL, root = NULL, locate = TRUE) {
     }
   }
   workdir <- file.path(path_draft(config$root), name, id)
-  recipe_commit(workdir, config$root)
+  
+  capture <- inherits(root, "orderly_config") && 
+    isTRUE(root$get_run_option("capture_log"))
+  logfile <- file.path(path_draft(config$root), name, id, "orderly.log")
+  conditional_capture_log(capture, logfile, recipe_commit(workdir, config$root))
```

Comment:
I think this will overwrite the actual run log?

Reply:
Should be okay now we have the append mode when opening the file connection. This is tested too https://github.com/vimc/orderly/blob/16a58d1d01190b4d85edd727f022bde97257aac5/tests/testthat/test-run.R#L1010

Reply:
sorry, I'd meant to delete that comment once I realised we're appending!